### ISSUE TITLE
[OMP][OJS] create ORCID-based SSO authentication service

### ISSUE THREAD

**Issue by crism** [pkp/pkp-lib#1451]
Dependent on #1450, once the ORCID plugin (from #1416) is working with the current stack, adapt it to be an authentication plugin as well.

**Comment by ctgraham** [comment]
I can be assigned this issue.

**Comment by mtub** [comment]
This will be so good to have…

**Comment by dennmuel** [comment]
Hello everyone,
are there any news for this feature request? Would be a really sweet one. =)
Kind regards and many thanks for all your great work!

**Comment by ctgraham** [comment]
I think this issue is now superseded by https://github.com/pkp/pkp-lib/issues/2818

**Comment by dennmuel** [comment]
Ah, I didn't see that one, sorry. Thanks for the quick response :)

**Comment by bozana** [comment]
Maybe I will then close this one...
